pal/audio/native/player.ts

```diff
@@ -62,7 +62,15 @@ export class AudioPlayer implements OperationQueueable {
      * @deprecated since v3.5.0, this is an engine private interface that will be removed in the future.
      */
     public _operationQueue: OperationInfo[] = [];
-
+    get sampleRate () :number {
+        if (!this._isValid) {
+            return 0;
+        }
+        return jsb.AudioEngine.getSampleRate(this._url);
```

Comment:
```suggestion
        return audioEngine.getSampleRate(this._url);
```

we can access the local audioEngine

Reply:
And as for the code style, I suggest to put the getter together, instead of inserting into all these public methods

Reply:
I'll revert the js modifications in the end.